1_Installation_Files (v1.5.5)/YOUR_ADMIN/includes/init_includes/init_image_handler.php

```diff
@@ -272,9 +272,17 @@
+                        ( 'Cache File-naming Convention', 'IH_CACHE_NAMING', '$default', 'Choose the method that <em>Image Handler</em> uses to name the resized images in the <code>bmz_cache</code> directory.<br /><br />The <em>Hashed</em> method was used by Image Handler versions prior to 4.3.4 and uses an &quot;MD5&quot; hash to produce the filenames.  It can be &quot;difficult&quot; to visually identify the original file using this method.  If you are upgrading Image Handler from a version prior to 4.3.4 <em>and</em> you have hard-coded links in product (or other) definitions to those images, <b>do not change</b> this setting from <em>Hashed</em>.<br /><br />Image Handler v4.3.4 (unreleased) introduced the concept of a <em>Readable</em> name for those resized images.  This is a good choice for new installations of <em>IH</em> or for upgraded installations that do not have hard-coded image links.<br /><br />Image Handler v5.1.9 introduced <em>Mirrored</em> which is the same as readable but it creates a directory structure under bmz_cache that mirrors the directory structure where the original images are saved.', $cgi, 1006, now(), NULL, 'zen_cfg_select_option(array(\'Hashed\', \'Mirrored\', \'Readable\'),')"
                 );
             }
+        } elseif (version_compare(IH_VERSION, '5.1.9', '<')) {
+            $db->Execute(
+                "UPDATE " . TABLE_CONFIGURATION . "
+                     SET configuration_description = 'Choose the method that <em>Image Handler</em> uses to name the resized images in the <code>bmz_cache</code> directory.<br /><br />The <em>Hashed</em> method was used by Image Handler versions prior to 4.3.4 and uses an &quot;MD5&quot; hash to produce the filenames.  It can be &quot;difficult&quot; to visually identify the original file using this method.  If you are upgrading Image Handler from a version prior to 4.3.4 <em>and</em> you have hard-coded links in product (or other) definitions to those images, <b>do not change</b> this setting from <em>Hashed</em>.<br /><br />Image Handler v4.3.4 (unreleased) introduced the concept of a <em>Readable</em> name for those resized images.  This is a good choice for new installations of <em>IH</em> or for upgraded installations that do not have hard-coded image links.<br /><br />Image Handler v5.1.9 introduced <em>Mirrored</em> which is the same as readable but it creates a directory structure under bmz_cache that mirrors the directory structure where the original images are saved.',
```

Comment:
This section should be placed towards the bottom of the initialization file, just under the update for v5.1.2, as a standalone `if` statement.  Please include lead-in commentary identifying the GitHub issue associated with the update.

Reply:
OK will move to make separate. Not used to git yet. Do i make the changes on my original version and then submit a new change request?

Reply:
You'll just update the changed version you have and re-submit the PR.

Reply:
Thanks for doing this, BTW.  I really appreciate it!